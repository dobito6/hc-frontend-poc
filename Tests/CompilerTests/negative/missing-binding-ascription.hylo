//! no-std

public fun main() {
  let x
}
