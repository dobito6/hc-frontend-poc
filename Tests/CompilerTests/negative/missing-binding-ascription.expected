negative/missing-binding-ascription.hylo:4.3-6: error: binding declaration requires an ascription
  let x
  ~~~
